test/index.js

```diff
@@ -343,6 +343,48 @@ describe('composition', function() {
 
   });
 
+  describe('converge', function() {
+
+    it('is a quaternary function', function() {
+      eq(typeof S.converge, 'function');
+      eq(S.converge.length, 4);
+    });
+
+    it('it transforms the value and passes the results to the first function',
+       function() {
+       eq(S.converge(S.and, S.head, S.at(2),
+                     ['a', 'b', 'c', 'd']), S.Just('c'));
+       eq(S.converge(S.and, S.head, S.at(10),
+                     ['a', 'b', 'c', 'd']), S.Nothing());
```

Comment:
Currently we have:

    eq(XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
                  XXXXXXXXXXXXXXXXXXXXX, YYYYYYYYYYY);

Instead, I would like to see:

    eq(XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX,
       YYYYYYYYYYY);

I find the nesting more apparent when it's reinforced by the indentation.